src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -137,19 +161,204 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
+                new ItemStack(Blocks.PLANKS, 1, META_FOUR), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(4), FOUR_BLOCKS, META_FOUR ), "SSS", "SXS", "SBS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FOUR), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(4), FOUR_BLOCKS, META_FIVE), "X  ", " X ", " BX", 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FOUR), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(4), FOUR_BLOCKS, META_SIX), "  X", " X ", "XB ", 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FOUR), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(4), FOUR_BLOCKS, META_SEVEN), "SXS", "SBS", "SSS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FOUR), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(4), FOUR_BLOCKS, META_EIGHT),"SSS", "XXX", "SBS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FOUR), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(4), FOUR_BLOCKS, META_NINE), "   ", "XXX", " B ", 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FOUR), 'B', ModItems.buildTool));
+
+
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(5), FOUR_BLOCKS, META_ZERO), " X ", " X ", " XB", 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(5), FOUR_BLOCKS, META_ONE), "X X", " X ", "XBX", 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(5), FOUR_BLOCKS, META_TWO), "SXS", "XBX", "SXS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(5), FOUR_BLOCKS, META_THREE), "SXS", "SXS", "SBS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(5), FOUR_BLOCKS, META_FOUR ), "SSS", "SXS", "SBS", 'S', WOODEN_STICK, 'X',
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)